client/src/Movies/Movie.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
+import { Route } from 'react-router-dom';
+import { Link } from 'react-router-dom';
 import axios from 'axios';
+import App from '../App.js'
+import MovieList from '../Movies/MovieList';
 
 export default class MovieCard extends React.Component {
   
   state = {
     movie: null,
   };
-
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const id = this.props.match.params.id;
```

Comment:
You can use object destructuring here: `const { id } = this.props.match.params;`